site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -487,6 +487,28 @@ public function updateAnnouncement() {
         );
     }
 
+
+    /**
+     * @Route("/courses/{_semester}/{_course}/office_hours_queue/stats", methods={"GET"})
+     * @return MultiResponse
+     */
+    public function showQueueStats() {
```

Comment:
You can just return a response object directly instead of using the static constructs from `MultiResponse` since they've been deprecated.

```php
 return new RedirectResponse(...)
    return new WebResponse(...)
```

will work fine